services/migration_service/migration_files/20201002000616_update_metadata_primary_key.sql

```diff
@@ -0,0 +1,43 @@
+-- +goose Up
```

Comment:
can you also make sure that migrations apply successfully in development mode, where metadata service has possibly created tables before migration service runs.

Reply:
changes in this migration file would still work even if the db was initially created by the metadata service